ouroboros-network/test/Test/Ouroboros/Network/PeerSelection/RootPeersDNS.hs

```diff
@@ -283,22 +332,23 @@ mockPublicRootPeersProvider :: forall m.
                             -> MockRoots
                             -> Script DNSTimeout
                             -> Script DNSLookupDelay
-                            -> Int
-                            -> m (Set SockAddr, DiffTime)
-mockPublicRootPeersProvider tracer (MockRoots localRootPeers dnsMap)
-                            dnsTimeoutScript dnsLookupDelayScript n = do
+                            -> ((Int -> m (Set SockAddr, DiffTime)) -> m a)
+                            -> m ()
+mockPublicRootPeersProvider tracer (MockRoots _ _ publicRootPeers dnsMap)
```

Comment:
Just a comment: if we use named field syntax, we could avoid this kind of changes.